src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/security/WebClientConfiguration.java

```diff
@@ -59,14 +57,10 @@ WebClient elite2ApiHealthWebClient() {
     @Bean
     public OAuth2AuthorizedClientManager authorizedClientManager(
             final ClientRegistrationRepository clientRegistrationRepository,
-            final OAuth2AuthorizedClientRepository authorizedClientRepository) {
+            final OAuth2AuthorizedClientService authorizedClientService) {
 
-        final OAuth2AuthorizedClientProvider authorizedClientProvider =
-                OAuth2AuthorizedClientProviderBuilder.builder().clientCredentials().build();
-
-        final DefaultOAuth2AuthorizedClientManager authorizedClientManager =
-                new DefaultOAuth2AuthorizedClientManager(clientRegistrationRepository, authorizedClientRepository);
-        authorizedClientManager.setAuthorizedClientProvider(authorizedClientProvider);
+        final OAuth2AuthorizedClientManager authorizedClientManager =
+                new AuthorizedClientServiceOAuth2AuthorizedClientManager(clientRegistrationRepository, authorizedClientService);
 
```

Comment:
The actual problem was using the wrong OAuth2AuthorizedClientManager - the "DefaultOAuth2AuthorizedClientManager" expects to be in a 'servlet context' which for events we are not.